.github/workflows/build_and_deploy.yml

```diff
@@ -0,0 +1,23 @@
+name: Build and deploy # Note: Deployment will be added in a future PR 
+
+on:
+  push:
+    branches-ignore:
+      - develop
+      - master
+      - support/**
+      - hotfix/**
+      - feature/**-i18n
+      - release/**
+
+jobs:
+  call_build:
+    uses: yext/answers-search-ui/.github/workflows/build.yml@dev/build-workflow
```

Comment:
is it okay that this is always pointing to `dev/build-workflow`?

Reply:
No, but the PM for Github Actions reusable workflows said that it will soon be possible to have it point to the same branch: https://github.community/t/ref-head-in-reusable-workflows/203690 Since we aren't in a rush to complete this migration, I was thinking we could hold off on completing the migration until the feature is released. Alternatively we could point the workflows at the feature branch in the mean time